CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 * Visual table summary? Want to know what's the most of something? No problem! You can acquire terms and statistics by just throwing attribute names into filter! Have a look at *Visual Table Summary* in this [UX Collective article](https://uxdesign.cc/design-better-data-tables-4ecc99d23356).
   * TO FUCKING DO!
```

Comment:
delete me...

Reply:
der pull reqeust geht ja nur in den 1.6.0-SNAPSHOT branch, da fehlt noch doku...